client/chain-spec/Cargo.toml

```diff
@@ -13,15 +13,30 @@ readme = "README.md"
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
+codec = { package = "parity-scale-codec", version = "3.6.1", default-features = false, features = ["derive"] }
+json-patch = { version = "1.0.0", default-features = false }
```

Comment:
Note: Added new dep: [json-patch](https://crates.io/crates/json-patch/0.2.0/dependencies).

Not sure if we need new crate for few-lines function.
I had a copy for no-std here:
https://github.com/paritytech/substrate/blob/df16a3d58455632f425761adeefb3105a5c2c07a/client/genesis-builder/src/lib.rs#L71-L93